pkgs/applications/science/misc/netlogo/default.nix

```diff
@@ -0,0 +1,35 @@
+{ jre, stdenv, fetchurl, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "NetLogo-${version}";
+  version = "6.0.4";
+
+  src = fetchurl {
+    url = "https://ccl.northwestern.edu/netlogo/${version}/NetLogo-${version}-64.tgz";
+    sha256 = "0dcd9df4dfb218826a74f9df42163fa588908a1dfe58864106936f8dfb76acec";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -pv $out/share/netlogo
+    cp -rv app readme.md $out/share/netlogo
```

Comment:
I don't think this `readme.md` will be useful in the derivation.

Reply:
To be honest, I'm not sure whether that README has to be included because it documents which libraries are used within the application?

Reply:
It is some useful documentation, so maybe it can go to `$out/share/doc`.